src/features/offer/components/OfferBody/OfferBody.tsx

```diff
@@ -148,6 +153,9 @@ export const OfferBody: FunctionComponent<Props> = ({ offerId, onScroll }) => {
     (value) => value !== undefined && value !== null
   )
 
+  const hasSameCategorySimilarOffers = sameCategorySimilarOffers?.length
+  const hasOtherCategoriesSimilarOffers = otherCategoriesSimilarOffers?.length
```

Comment:
Effectivement `undefined > 0` ça marcherait pas :P

Dans ce cas je préfère mieux
```suggestion
  const hasSameCategorySimilarOffers = Boolean(sameCategorySimilarOffers?.length)
  const hasOtherCategoriesSimilarOffers = Boolean(otherCategoriesSimilarOffers?.length)
```

parce que là niveau TypeScript tu dois avoir un truc du genre `number | undefined` alors qu'on veut un boolean

Reply:
Yes c'est vrai 👍

Reply:
Sur `OfferBody` ça me génère une erreur Typescript au niveau du param `data` de `PassPlaylist` :

> Impossible d'assigner le type 'SearchHit[] | undefined' au type 'any[]'.
>   Impossible d'assigner le type 'undefined' au type 'any[]'.